src/index.html

```diff
@@ -15,8 +15,101 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="img_link"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+          class="img"
+        />
+      </a>
+
+      <nav class="nav_header">
+        <ul class="nav_bar">
+          <li class="nav_list">
+            <a
+              href="#"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_list">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav_list">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav_list">
+            <a
+              href="#"
+              class="nav_link blue"
```

Comment:
```suggestion
              class="nav_link"
```
we don't need this class, try to rewrite without it